docs/scarpet/api/Entities.md

```diff
@@ -801,6 +801,7 @@ defining the callback with `entity_event`.
 The following events can be handled by entities:
 
 *   `'on_tick'`: executes every tick right before the entity is ticked in the game. Required arguments: `entity`
+*   `'on_move'`: executes every time an entity changes position, invoked just after it has been moved to the new position. Required arguments: `entity, velocity, pos1, pos2`
```

Comment:
Suggest renaming `pos1`and `pos2` to `prev_pos` and `new_pos` to make it more explicit?